manifests/j/JohnMacFarlane/Pandoc/3.6.2/JohnMacFarlane.Pandoc.installer.yaml

```diff
@@ -0,0 +1,35 @@
+# Created with WinGet Updater using komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: JohnMacFarlane.Pandoc
+PackageVersion: 3.6.2
+InstallerLocale: en-US
+InstallerType: wix
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: uninstallPrevious
+ProductCode: '{C09543E4-2E4C-4F2C-8688-EA6E43F9DBE5}'
+ReleaseDate: 2025-01-12
+AppsAndFeaturesEntries:
+- DisplayName: Pandoc 3.6.2
+  Publisher: John MacFarlane
+  ProductCode: '{C09543E4-2E4C-4F2C-8688-EA6E43F9DBE5}'
+  UpgradeCode: '{A68E8EF6-ABB1-4F22-A3C5-68DFDF0AB562}'
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%/Pandoc'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/jgm/pandoc/releases/download/3.6.2/pandoc-3.6.2-windows-x86_64.msi
+  InstallerSha256: E9A8036CC298AE3D92105D5909C805A0A069CC988E7938703C4041B3E24A13AA
+  InstallerSwitches:
+    Custom: ALLUSERS=1
+- Architecture: x64
+  InstallerUrl: https://github.com/jgm/pandoc/releases/download/3.6.2/pandoc-3.6.2-windows-x86_64.msi
+  InstallerSha256: E9A8036CC298AE3D92105D5909C805A0A069CC988E7938703C4041B3E24A13AA
+  InstallerSwitches:
+    Custom: ALLUSERS=2

```

Comment:
This entry has a different custom switch - what sort of install does `ALLUSERS=2` result in? Should `user` and `machine` `Scope` be used here, to better differentiate these entries?